modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -107,17 +121,19 @@ export default class TileLayer extends CompositeLayer {
     };
   }
 
-  _updateTileset() {
-    const {tileset} = this.state;
+  _updateTileset(tileset, reload = false) {
```

Comment:
Open to just setting up a new method but it feels like it would need to look similar i.e change frame number, check loading, call `onViewportLoad` etc.